package.json

```diff
@@ -1,14 +1,23 @@
 {
   "name": "gtoken",
-  "version": "1.2.3",
+  "version": "2.0.0",
   "description": "Node.js Google Authentication Service Account Tokens",
-  "main": "./lib/index.js",
+  "main": "build/src/index.js",
+  "types": "./build/src/index.d.ts",
```

Comment:
nit: Use consistent style between `main` and `types`.

Reply:
fixed